requirements.txt

```diff
@@ -9,6 +9,7 @@ pyproj==2.6.1.post1
 python-dateutil==2.7.5
 requests~=2.24.0
 scipy==1.4.1
+tox==3.23.1
```

Comment:
I would leave this out since tox is usually part of the global environment - this avoids some redundant installation and enables running `tox` whether or not a virtualenv is activated.

Reply:
Thanks! Fixed